<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementEditor</class>
 <widget class="QWidget" name="ElementEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFrequency">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_frequency">
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditElementLabel"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_trigger">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_inputs">
     <property name="text">
      <string>Input Ports:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="comboBoxInputSz"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_labels">
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="comboBoxColor">
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_color">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="lineEditTrigger"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditElementLabel</tabstop>
  <tabstop>comboBoxInputSz</tabstop>
  <tabstop>doubleSpinBoxFrequency</tabstop>
  <tabstop>comboBoxColor</tabstop>
  <tabstop>lineEditTrigger</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
